int32 offbCtlType
---
bool success